pkg/reconciler/kubernetes/tektonresult/tektonresult.go

```diff
@@ -298,6 +302,59 @@ func (r *Reconciler) ReconcileKind(ctx context.Context, tr *v1alpha1.TektonResul
 	return nil
 }
 
+// This function exists to resolve conflict between PreReconciler and Reconciler Installerset.
+// This won't be needed in the next release of operator.
+func (r *Reconciler) conflictTIS(ctx context.Context) error {
+	logger := logging.FromContext(ctx)
+	// Check if an tektoninstallerset already exists, if not then create
+	labelSelector, err := common.LabelSelector(ls)
+	if err != nil {
+		return err
+	}
+	existingInstallerSet, err := tektoninstallerset.CurrentInstallerSetName(ctx, r.operatorClientSet, labelSelector)
```

Comment:
@khrm this can be issue after upgrade too.

We will be running this again and again. After upgrade there will be two installerset. And this function will delete both. and That will keep happening on loop.

Reply:
Why will there be two installerset? Only one will remain there.

Reply:
did you mention that there will be two installerset after this - one of db and one of results

Reply:
That other one won't be available with this. It's a pre-installer set. It's labelSelector is different.

Reply:
aah ohk, the value of type will be different

Reply:
Thanks. :)